docs/running-clusterfuzzlite/github_actions.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: default
-parent: Running ClusterFuzzLite
+parent: Step 2&#58; Running ClusterFuzzLite
```

Comment:
What does `&#58` do exactly?

Reply:
It's the HTML encoding of a colon.
For some reason putting a colon in the title without the > broke things completely. But the > seemed to break things too.